std/meta.d

```diff
@@ -863,6 +863,95 @@ unittest
     }
 }
 
+private template staticMerge(alias C, int half, T...)
```

Comment:
Private helper functions should go below what they help. Going above comes from C/C++ where things have to be defined before use - D doesn't have that limitation.